src/Mapbender/CoreBundle/Element/Type/OverviewAdminType.php

```diff
@@ -29,6 +29,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('layerset', 'Mapbender\CoreBundle\Element\Type\LayersetAdminType', array(
                 'application' => $options['application'],
                 'required' => true,
+                'label' => 'mb.core.overview.admin.layerset',
```

Comment:
wrong: 'mb.core.overview.admin.layerset',
should be: 'mb.core.layerset.admin.layerset',